gdal/swig/python/scripts/gdal_edit.py

```diff
@@ -303,6 +323,11 @@ def gdal_edit(argv):
               uly, 0, (lry - uly) / ds.RasterYSize]
         ds.SetGeoTransform(gt)
 
+    if lly is not None:
```

Comment:
To make it clear that lry and lly are exclusive:
```suggestion
    elif lly is not None:
```